helm-values/traction/values-sandbox.yaml

```diff
@@ -0,0 +1,85 @@
+ingressSuffix: .apps.silver.devops.gov.bc.ca
+acapy:
+  image:
+    pullPolicy: Always
+  secret:
+    adminApiKey:
+      generated: true
+    pluginInnkeeper:
+      generated: true
+  plugin-config.yml:
+    traction_innkeeper:
+      innkeeper_wallet:
+        print_key: true
+        print_token: true
+        connect_to_endorser:
+          - endorser_alias: endorser
+            ledger_id: bcovrin-test
+        create_public_did:
+          - bcovrin-test
+      reservation:
+        expiry_minutes: 2880
+        auto_approve:  true
+        auto_issuer: true
+      autoscaling:
```

Comment:
This is not indented correctly: as it is right now, it will be considered a child item of `plugin-config.yml`, it should instead be a sibling at the same level.